new-lamassu-admin/src/pages/Dashboard/Footer/Footer.js

```diff
@@ -38,7 +37,7 @@ const Footer = () => {
   const withCommissions = R.path(['cryptoRates', 'withCommissions'])(data) ?? {}
   const classes = useStyles()
   const config = R.path(['config'])(data) ?? {}
-  const canExpand = R.keys(withCommissions).length > 4
+  // const canExpand = R.keys(withCommissions).length > 4
```

Comment:
Is this intentional?

Reply:
Variable hasn't been used for a long time, I'll just remove it